.gitignore

```diff
@@ -1,4 +1,4 @@
-engine/
```

Comment:
It is convenient for windows, because automatic download is not supported, which does not affect other platforms.